src/util/rocksdb/engine_metrics.rs

```diff
@@ -523,6 +523,20 @@ lazy_static!{
             &["type"]
         ).unwrap();
 
+    pub static ref STORE_ENGINE_COMPACTION_DURATIONS_VEC: HistogramVec =
+        register_histogram_vec!(
+            "tikv_engine_compaction_duration_seconds",
+            "Histogram of compaction duration seconds",
+            &["cf"]
```

Comment:
the default histogram bucket max time is only 10s, maybe too small, we should use our own buckets.